content/training/Walk-through/univariate.Rmd

```diff
@@ -138,7 +138,7 @@ tab_linelist(age_group, sex, strata = case_def, na.rm = FALSE) %>%
   ## run chi-sq test

```

Comment:
sorry this is the first place i could comment but in line 103 - it calls a vector of chars a dataframe... just for consistency

Reply:
and again in line 107  - and needs a second hashtag

Reply:
I think I might have fixed these?

Reply:
they are still there Zhian

Reply:
Just checked. The only instances that use "data frame" are indeed data frames

Reply:
OK. Good.